theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -314,11 +317,12 @@ def fn(x):
             # Test the GPU grad + GPU implementation
             def fn(x):
                 dnn_op = cuda.dnn.dnn_pool(
-                    x, ws=(ws, ws),
-                    stride=(stride, stride),
-                    pad=pad,
+                    x, ws=ws,
+                    stride=stride,
+                    pad=pad_,
```

Comment:
Why this change? Do it work with the old code? It should work to not break too much compatibility.